code/modules/medical/surgery_tools.dm

```diff
@@ -47,18 +47,19 @@ CONTAINS:
 		if (src.icon_state == "scalpel1")
 			icon_state = pick("scalpel1", "scalpel2")
 		src.create_reagents(5)
+		BLOCK_KNIFE
 
 	attack(mob/living/carbon/M as mob, mob/user as mob)
 		if (src.reagents && src.reagents.total_volume)
 			logTheThing("combat", user, M, "used [src] on %target% (<b>Intent</b>: <i>[user.a_intent]</i>) (<b>Targeting</b>: <i>[user.zone_sel.selecting]</i>) [log_reagents(src)]")
-			src.reagents.trans_to(M,5-(5*0.75*(min(M.get_melee_protection(user.zone_sel.selecting)/min(src.force,1),1))))
+			src.reagents.trans_to(M,5-(5*0.75*(min(M.get_melee_protection(user.zone_sel.selecting, src.hit_type)/min(src.force,1),1))))
```

Comment:
Surgical tools applying chems on attack is handled through the `transfer_on_attack` component. You shouldn't need to be poking this stuff here